.report/20260731_#466_배터리_제한없음_없이_백그라운드_유지.md

```
@@ -0,0 +1,81 @@
+# [#466] 배터리 "제한없음" 설정 없이 백그라운드 게임 유지
+
+### 📌 작업 개요
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)